src/lib.rs

```diff
@@ -10,7 +10,7 @@ mod cli;
 mod config;
 mod database;
 mod defaults;
-mod transport;
+pub mod transport;
```

Comment:
At some point it seemed to matter, but that might have been some intermediary state. I'll remove it.

Reply:
Ah, I remember, it is needed because the `ZkChannelAddress` is used inside the documentation, hence, it needs to be public. Note that the transport mentioned here is merely that object and not much else (everything contained in `transport.rs`.

Reply:
I see, that seems reasonable. `ZkChannelAddress` is something that external users might have to deal with anyway (e.g. to format their RPC requests eventually).